TokenManagementSystem/app/Participant.php

```diff
@@ -9,7 +9,7 @@ class Participant extends Model
     protected $guarded = []; //allowing mass assignment
 
     public function student(){
-    	return $this->hasOne('\App\Students');
+    	return $this->hasOne('\App\Students',$foreignKey="id",$ownerKey="student_id");
```

Comment:
Check if these works. Cause here local key should come. i.e. foreignkey=student_id and localkey=id

Reply:
Yes I have checked it and it works
@Rohan-RPJ

Reply:
hasOne is not applicable in student.php because a student can have multiple participants as they're created for each submission. But every participant will have one student and hence why I used this relationship